docs/source/administrator/services.md

```diff
@@ -18,11 +18,15 @@ In the following snippet, I'm using a custom image that copies over the applicat
 
 ```Dockerfile
 # Dockerfile
-# 0.11.1 is latest stable release at the time of this writing
-FROM jupyterhub/k8s-hub:0.11.1
+# 2.0.0 is latest stable release at the time of this writing
+FROM jupyterhub/k8s-hub:2.0.0
```

Comment:
We've got `chart_version` as a variable
https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/29b041ced53678357d062c188f52e8983bfb1f03/docs/source/conf.py#L71-L72

@consideRatio Do you know of a simple way we could get `last_released_chart_version` to avoid having to keep this up to date?

Reply:
Hmmmm, I recall there is a complexity around having a substitution inside a triple-backtick example. So, there may be two aspects to consider here.

But getting the latest released chart version is something I typically like getting done via `helm` and `yq`, which isn't available everywhere without trouble.

```
helm show chart --repo=https://hub.jupyter.org/helm-chart jupyterhub | yq eval ".version" -
```

For our charts specifically, we can use the `info.json` file we host alongside the Helm required `index.yaml` that summarizes the latest stable version of the charts: https://hub.jupyter.org/helm-chart/info.json, that way all we would need is a web request.